web/static/js/app.js

```diff
@@ -2,6 +2,12 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import AllHabits from './components/AllHabits'
 import App from './components/App'
+import Promise from 'promise-polyfill';
+
+// To add to window
+if (!window.Promise) {
+  window.Promise = Promise;
+}
```

Comment:
The only snag is that the latest Phantomjs doesn't yet support `Promise`s, even though most modern browsers do. So I needed to add this polyfill which is probably a good idea for legacy browser support anyway.